doc/example/prior_definition.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Definition of Priors:\n",
+    "\n",
+    "In this notebook we demonstrate, how to define (log-)priors for parameters. If you want to maximize your posterior distribution, you need to define\n",
```

Comment:
```suggestion
    "In this notebook we demonstrate how to define (log-)priors for parameters. If you want to maximize your posterior distribution, you need to define\n",
```

Reply:
it should not only be about "maximizing" the posterior, but about performing ifnerence using prior knowledge combined with the likelihood of data given parameters.